vsintegration/tests/UnitTests/ProjectDiagnosticAnalyzerTests.fs

```diff
@@ -20,15 +20,15 @@ open Microsoft.VisualStudio.FSharp.LanguageService
 open FSharp.Compiler.SourceCodeServices
 open FSharp.Compiler.Range
 
-[<TestFixture>][<Category "Roslyn Services">]
-type ProjectDiagnosticAnalyzerTests()  =
+//[<TestFixture>][<Category "Roslyn Services">]
+//type ProjectDiagnosticAnalyzerTests()  =
 
-    let CreateProjectAndGetOptions(fileContents: string) =
-        let tempName = Path.GetTempFileName()
-        let fileName = Path.ChangeExtension(tempName, ".fs")
-        let projectName = Path.ChangeExtension(tempName, ".fsproj")
-        let dllName = Path.ChangeExtension(tempName, ".dll")
-        File.WriteAllText(fileName, fileContents)
+//    let CreateProjectAndGetOptions(fileContents: string) =
+//        let tempName = Path.GetTempFileName()
+//        let fileName = Path.ChangeExtension(tempName, ".fs")
+//        let projectName = Path.ChangeExtension(tempName, ".fsproj")
+//        let dllName = Path.ChangeExtension(tempName, ".dll")
+//        File.WriteAllText(fileName, fileContents)
 
-        let args = mkProjectCommandLineArgs (dllName, [fileName])
-        checker.GetProjectOptionsFromCommandLineArgs (projectName, args)
+//        let args = mkProjectCommandLineArgs (dllName, [fileName])
+//        checker.GetProjectOptionsFromCommandLineArgs (projectName, args)
```

Comment:
Why is this commented out?

Reply:
This code is unused (and doesn't compiler) once the "service" layer tests got moved to FSharp.Compiler.UnitTests.  I'll remove the whole file.  It doesn't actually test anything